tests/testing/LocalBearTestHelperTest.py

```diff
@@ -101,8 +104,44 @@ def setUp(self):
         section.append(Setting('exception', True))
         self.uut = TestBear(section, Queue())
 
-    def test_exception(self):
+    def test_stdout_stderr_on_linter_test_fail(self):
+        process_output_mock = Mock()
+
+        class TestLinter:
+            @staticmethod
+            def process_output(output, filename, file):
+                process_output_mock(output, filename, file)
+
+            @staticmethod
+            def create_arguments(filename, file, config_file):
+                code = '\n'.join(['import sys',
+                                  "print('hello stdout')",
+                                  "print('hello stderr', file=sys.stderr)"])
+                return '-c', code
+
+        # Testing with both stdout and stderr enabled
+        uut = (linter(sys.executable, use_stdout=True, use_stderr=True)
+               (TestLinter)
+               (Section('TEST_SECTION'), Queue()))
+        try:
+            with execute_bear(uut, 'filename', ['file']) as result:
+                self.assertNotEqual(result, [])
+        except AssertionError as ex:
+            self.assertIn('hello stdout', str(ex))
+            self.assertIn('hello stderr', str(ex))
+
+        # Testing with only stdout enabled
+        uut = (linter(sys.executable, use_stdout=True)
+               (TestLinter)
+               (Section('TEST_SECTION'), Queue()))
+        try:
+            with execute_bear(uut, 'filename', ['file']) as result:
+                self.assertNotEqual(result, [])
+        except AssertionError as ex:
+            self.assertIn('hello stdout', str(ex))
+            self.assertNotIn('hello stderr', str(ex))
 
+    def test_exception(self):
         with self.assertRaises(AssertionError), execute_bear(
-                self.uut,  'Luke', files[0]) as result:
+                self.uut, 'Luke', files[0]) as result:
```

Comment:
1. unnecessary change
2. did something change in the line with `def test_exception(self):`? Wondering why the diff removes it above and adds it here again :O